### PR TITLE
Add copy and restore

### PR DESCRIPTION
**Pull request by chregu**

### CODE REVIEW

File: src/apis/sourceimages.js [hunk 1]
```diff
@@ -236,6 +236,49 @@ export default (state) => {
...
+   * rokka.sourceimages.copy('myorg', 'c421f4e8cefe0fd3aab22832f51e85bacda0a47a', 'anotherorg', true)
+   *   .then(function(result) {})
+   *   .catch(function(err) {});
+   * ```
+   *
+   * @authenticated
+   * @param  {string}  organization            the org the image is copied from
+   * @param  {string}  hash                    image hash
+   * @param  {string}  destinationOrganization the org the image is copied to
+   * @param  {boolean} [overwrite = false]     if an existing image should be overwritten
```
Comment (mweibel) [inline src/apis/sourceimages.js]: the default is `true` ;)

File: src/apis/sourceimages.js [hunk 2]
```diff
@@ -236,6 +236,49 @@ export default (state) => {
...
+   *
+   * @return {Promise}
+   */
+  sourceimages.copy = (organization, hash, destinationOrganization, overwrite = true) => {
+    const headers = {'Destination': destinationOrganization}
+    if (overwrite === false) {
```
Comment (mweibel) [inline src/apis/sourceimages.js]: I'd just shorten this to `!overwrite`

File: src/apis/sourceimages.js [hunk 3]
```diff
@@ -236,6 +236,49 @@ export default (state) => {
...
+      headers['Overwrite'] = 'F'
+    }
+    return state.request('COPY', `sourceimages/${organization}/${hash}`, null, null, {headers: headers})
```
Comment (mweibel) [inline src/apis/sourceimages.js]: `{ headers }` is enough :)

**Comment by chregu** [comment]
Thanks, should all be fixed
